package.json

```diff
@@ -192,6 +216,88 @@
           "meta.embedded.block.rego": "rego"
         }
       }
+    ],
+    "breakpoints": [
+      {
+        "language": "rego"
+      }
+    ],
+    "debuggers": [
+      {
+        "type": "opa-debug",
+        "label": "OPA Debug",
+        "languages": [
+          "rego"
+        ],
+        "configurationAttributes": {
+          "launch": {
+            "properties": {
+              "command": {
+                "type": "string",
+                "description": "The OPA command to run. E.g. 'eval', 'test'.",
+                "default": "run"
+              },
+              "query": {
+                "type": "string",
+                "default": "data"
+              },
+              "dataPaths": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "bundlePaths": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "default": ["${workspaceFolder}"]
+              },
+              "input": {
+                "type": "string"
+              },
+              "inputPath": {
+                "type": "string"
+              },
+              "stopOnEntry": {
+                "type": "boolean",
+                "description": "Automatically stop after launch.",
+                "default": true
+              },
+              "stopOnFail": {
+                "type": "boolean",
+                "description": "Automatically stop on 'Fail' operations.",
+                "default": false
+              },
+              "stopOnResult": {
+                "type": "boolean",
+                "description": "Automatically stop when done.",
+                "default": true
+              },
+              "trace": {
+                "type": "boolean",
+                "description": "Enable logging of the Debug Adapter Protocol.",
+                "default": true
+              },
+              "enablePrint": {
+                "type": "boolean",
+                "description": "Enable print statements.",
+                "default": true
+              },
+              "logLevel": {
+                "type": "string",
+                "description": "Set the log level. One of: 'debug', 'info', 'warn', 'error'."
```

Comment:
Log level of what exactly? :)
And what'd the default?

Reply:
The level of logging that is returned to the vs-code debug console. To make it possible to troubleshoot debugger issues. There is no default. If not set, no logging is returned at all. I'll update the description.

Reply:
Thanks!